### DISCUSSION TITLE
MathJax in py-shiny

### DISCUSSION THREAD

**Discussion by DTMhango** [posit-dev/py-shiny#1269] [General]
R-shiny has the withMathjax() function to render mathematical equations. Is there an equivalent way to do this in py-shiny? I can't seem to figure out how to do this.

**Comment by cpsievert** [comment]
There currently isn't an equivalent abstraction, but you can include HTML like this in the UI to do essentially the same thing

https://github.com/posit-dev/py-shiny-templates/blob/13612709/regularization/shared.py#L21-L27
